docs/components/base/sensor-controlled.md

```diff
@@ -73,6 +73,7 @@ The following attributes are available for `sensor-controlled` bases:
 | ---- | ---- | --------- | ----------- |
 | `movement-sensor` | array | **Required** | Array with the `name`s of any movement sensors on your base you want to gather feedback from. The driver will select the first movement sensor providing appropriate feedback for either the `SetVelocity()` or the `Spin()` endpoint. |
```

Comment:
Also noticed this typo: I think it should be `movement_sensor` (underscore not dash)